packages/genesys-spark-components/src/components/stable/gux-dropdown/gux-dropdown.tsx

```diff
@@ -531,10 +533,10 @@ export class GuxDropdown {
     return (
       <div
         class={{
-          'gux-target-container-expanded': this.expanded && this.isFilterable(),
-          'gux-target-container-collapsed': !(
-            this.expanded && this.isFilterable()
-          ),
+          'gux-target-container-filterable': this.isFilterable(),
+          'gux-target-container-filterable-active':
```

Comment:
Maybe its not idea to change these due to snapshot changes but it just made more sense to me. Let me know.